StripeFinancialConnections/StripeFinancialConnections/Source/Native/NetworkingLinkSignupPane/NetworkingLinkSignupDataSource.swift

```diff
@@ -68,14 +68,17 @@ final class NetworkingLinkSignupDataSourceImplementation: NetworkingLinkSignupDa
         emailAddress: String,
         phoneNumber: String,
         countryCode: String
-    ) -> Future<FinancialConnectionsSessionManifest> {
-        return apiClient.saveAccountsToLink(
+    ) -> Future<String?> {
+        return apiClient.saveAccountsToNetworkAndLink(
+            shouldPollAccounts: !manifest.shouldAttachLinkedPaymentMethod,
```

Comment:
[web logic for boolean on whether pooling should happen](https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/master/src/linkedAccounts/inner/hooks/v3/useSaveToNetworkAndLink.ts#L103)